in-progress/8754-slashing-and staking.md

```diff
@@ -0,0 +1,357 @@
+
+|                      |                                   |
+| -------------------- | --------------------------------- |
+| Issue                | [title](github.com/link/to/issue) |
+| Owners               | @aminsammara                              |
+| Approvers            | @LHerskind @just-mitch @joeandrews @Maddiaa0 |
+| Target Approval Date | 2024-11-08                        |
+
+
+# Executive Summary
+
+This design introduces a Staking Mechanism and Proof Of Governance (PoG) for slashing. 
+
+For Staking, we introduce an immutable `Deposit` contract to make stake migrations more seamless.  
+
+For Slashing, the rationale behind using PoG is the use of offchain evidence for voting allows nice properties including:
+
+1) More precision in slashing. Honest node operators should not be slashed but we need offchain evidence to do this. 
+2) Provers can appeal to the validator set to unslash them in the case they could not submit their proof on time (i.e. Ethereum congestion, or an attack on proving marketplace nodes)
+
+So we rely on validators to vote to slash dishoneset actors. 
+
+# Staking
+
+Validators stake with a `Deposit` contract in order to join the validator set. They do NOT stake with the Rollup contract directly. This is meant to simplify moving stake during/after a governance upgrade. 
+
+### Deposit Contract
+
+The Deposit contract is an immutable contract living on L1, that is owned by the Governance Contract (i.e. Apella). It implements the following simplified interface:
+
+```solidity
+
+interface IDeposit {
```

Comment:
nit: this interface could which of these functions are privileged, useful to have this context earlier rather than learning about it later on

e.g. activateValidator is only callable by a rollup contract.